README.md

```diff
@@ -169,9 +169,9 @@ The empty values can be also limited to `null` only:
 
 ```php
 use Yiisoft\Validator\Rule\Number;
-use Yiisoft\Validator\SkipOnEmptyCallback\SkipOnNull;
+use Yiisoft\Validator\EmptyHandler\NullEmpty;
```

Comment:
In my opinion `EmptyHandler` folder name is a bit confusing. Maybe use `EmptyValue` name or something like that?

Reply:
I think `EmptyHandler` is fine. Better than `EmptyCriteria` because it's not simple a crtiteria for emptiness but a handler i.e. `SkipOnNull` doesn't only determine what is "empty" but defines how to handle it.

Reply:
Hmm... checked the actual code. Seems it's, indeed, a crtiteria 🤔